packages/@cdktf/provider-generator/lib/get/generator/emitter/resource-emitter.ts

```diff
@@ -52,6 +55,14 @@ export class ResourceEmitter {
     );
   }
 
+  private emitStaticMethods(resource: ResourceModel) {
+    this.code.line(
+      `public static importGenerateConfig(scope: Construct, name: string, id: string, provider?: cdktf.TerraformProvider) {
```

Comment:
We should also emit a comment here that explains what this static call does and links to our import docs for more information.

Reply:
👍